src/lib/y2partitioner/actions/controllers/lvm_vg.rb

```diff
@@ -203,6 +203,27 @@ def remove_device(device)
           BlkDeviceRestorer.new(device.plain_device).restore_from_checkpoint
         end
 
+        # Whether the volume group can allocate each striped logical volume.
+        #
+        # Note that this method only checks that the volume group has the necessary physical volumes to
+        # allocate each striped volume individually. Checking that the volume group can allocate all the
+        # striped volumes at the same time would be very tricky. Such a check requires to know how the
+        # volumes are distributed in the physical volumes.
+        #
+        # @return [Boolean]
+        def size_for_striped_lvs?
+          striped_lvs.all? { |l| vg.size_for_striped_lv?(l.size, l.stripes) }
+        end
+
+        # Maximum number of stripes used by the striped logical volumes of the volume group
+        #
+        # @return [Integer] 0 if no striped volumes
+        def max_stripes
```

Comment:
Not sure if `max_stripes` is a good name. It sounds more like a limit than as a current value. Why not simply `stripes` or something slightly more explicit like `lvs_stripes`.